source/extensions/omni.isaac.orbit/omni/isaac/orbit/sim/utils.py

```diff
@@ -777,3 +777,47 @@ def find_global_fixed_joint_prim(
                 break
 
     return fixed_joint_prim
+
+
+"""
+USD Variants.
+"""
+
+
+def set_usd_variants(prim_path: str, variants: object | dict[str, str], stage: Usd.Stage | None = None) -> None:
```

Comment:
NIT: Can we rename this function to `select_usd_variants`? We try to avoid function with the name `set_` because it usually requires parity with a `get_`.